<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login Demo</string>
  </property>
  <widget class="QLabel" name="username_Label">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>52</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_Label">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>83</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_lineEdit">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>52</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_lineEdit">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>83</y>
     <width>125</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="user_textBrowser">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="login_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>115</y>
     <width>76</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_pushButton">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>115</y>
     <width>68</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Python</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Java</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C++</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ruby</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="openfile_pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chuang3bed_pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>221</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>YuMincho +36p Kana</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>åºŠ</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
